pages/index.html

```diff
@@ -1,5 +1,216 @@
 {% extends "base.html" %}
 
 {% block content %}
-future.m.o home page
+    <!-- Hero section -->
```

Comment:
this page should actually be at `pages/mieco/index.html` right? (or `pages/mieco.html` would also work).